tests/FakeItEasy.Specs/CreationSpecs.cs

```diff
@@ -177,5 +204,10 @@ protected override IList<T> CreateCollectionOfFake<T>(int numberOfFakes)
         {
             return A.CollectionOfFake(typeof(T), numberOfFakes).Cast<T>().ToList();
         }
+
+        protected override IList<T> CreateCollectionOfFake<T>(int numberOfFakes, Action<IFakeOptions<T>> optionsBuilder)
+        {
+            return A.CollectionOfFake(typeof(T), numberOfFakes, options => optionsBuilder((IFakeOptions<T>)options)).Cast<T>().ToList();

```

Comment:
The non-generic overload accepts an `Action<IFakeOptions>`, not `Action<IFakeOptions<T>>` There's no inheritance relation between the interfaces, so contravariance doesn't work here.